src/views/Scenario/Scenario.js

```diff
@@ -121,14 +127,13 @@ const Scenario = (props) => {
       </Grid>
       <Grid item xs={12}>
         <Card style={{ height: '400px' }}>
-          { currentScenario.data &&
             <Dashboard
               iframeTitle={t('commoncomponents.iframe.scenario.results.card.title', 'Results')}
-              url={SCENARIO_DASHBOARD_CONFIG.url.replaceAll('<ScenarioName>', currentScenario.data.name)}
-              scenarioName={currentScenario.data.name}
-              scenarioState={currentScenario.data.state}
+              url={SCENARIO_DASHBOARD_CONFIG.url.replaceAll('<ScenarioName>', currentScenario.data?.name)}
+              scenarioName={currentScenario.data?.name}
+              scenarioState={currentScenario.data?.state}
```

Comment:
What if currentScenario is undefined? is the `?` at the right place?

Reply:
cuurentSecnrio is set from a reducer and cannot be null